cpp/tests/hash_map/map_test.cu

```diff
@@ -68,10 +68,7 @@ struct InsertTest : public cudf::test::BaseFixture {
 
 using TestTypes = ::testing::Types<key_value_types<int32_t, int32_t>,
                                    key_value_types<int64_t, int64_t>,
-                                   key_value_types<int8_t, int8_t>,
                                    key_value_types<int16_t, int16_t>,
-                                   key_value_types<int8_t, float>,
-                                   key_value_types<int16_t, double>,
```

Comment:
Removed types not supported by `atomic_ref`. Not really an issue since those data structures will be replaced by cuco ones soon.